doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
+corresponding subscription, see subscriptions operations section for details).
+
+* attrsFormat ("full", "keyValues", "values"; default is "full") to specify how the attributes are
+  represented. See " Simplified Entity Representation" for details.
+* dataFormat ("array", "object"; default is "array"): to specify if the `data` field holding the information
+  is an array or an object.
+  * In "array" case, "data" is an array which elements are objects, each one representing an entity.
+  * In "object" case, "data" in an object which keys are entity types and which values are object which keys, in sequence,
+    are entity ids and which values is an object representing the entity.
+
+Combining both, there are 6 possibilities:
+
+* attrsFormat is "full" and "dataFormat" is "array":
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [
+     {
+       "id": "Room1",
+       "type": "Room",
+       "temperature": {
+         "value": 23,
+         "type": null,
+         "metadata": {}
+       },
+       "humidity": {
+         "value": 70,
+         "type": "percentage",
+         "metadata": {}
+       }
+     },
+     {
+     "id": "Room2",
+     "type": "Room",
+     "temperature": {
+       "value": 24,
+       "type": null,
+       "metadata": {}
+     }
+   ]
+}
+```
+
+* attrsFormat is "full" and "dataFormat" is "object":
+
+```
+{
+   "subscriptionId": "12345",
+   "data": {
+     "Room": {
+       "Room1": {
+         "temperature": {
+           "value": 23,
+           "type": null,
+           "metadata": {}
+         },
+         "humidity": {
+           "value": 70,
+           "type": "percentage",
+           "metadata": {}
+         }
+       },
+       "Room2": {
+         "temperature": {
+           "value": 24,
+           "type": null,
+           "metadata": {}
+         }
+       }
+     }
+   }
+}
+
+```
+
+* attrsFormat is "keyValues" and "dataFormat" is "array":
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [
+     {
+       "id": "Room1",
+       "type": "Room",
+       "temperature": 23,
+       "humidity": 70
+       }
+     },
+     {
+     "id": "Room2",
+     "type": "Room",
+     "temperature": 24
+     }
+   ]
+}
+```
+
+* attrsFormat is "keyValues" and "dataFormat" is "object":
+
+```
+{
+   "subscriptionId": "12345",
+   "data": {
+     "Room": {
+       "Room1": {
+         "temperature": 23,
+         "humidity": 70
+       },
+       "Room2": {
+         "temperature": 24
+       }
+     }
+   }
+}
+
+```
+
+* attrsFormat is "values" and "dataFormat" is "array":
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [ [23, 70], [24] ]
+}
+```
+
+* attrsFormat is "values" and "dataFormat" is "object":
+
+```
+{
+   "subscriptionId": "12345",
+   "data": {
+     "Room": {
+       "Room1": [23, 70],
+       "Room2": [24]
+     }
+   }
+}
+
+```
+
+### Notification templates
+
+Based on the "template" field defined as part of the subscription and in a macro replacement language
```

Comment:
@mrutid , a next PR will include a proposal for this (by the moment, let's reach agreement in the current content of the PR before addressing that next step).